mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = (req, res) => {
+  Order.remove({ _id: req.params.orderId }) 
```

Comment:
The `remove` method is deprecated in newer versions of MongoDB and should be replaced with `deleteOne` or `deleteMany` for better performance and compatibility. It's recommended to use `deleteOne` in this case since we're deleting a single order based on the provided `orderId` parameter.